chapters/annotations.tex

```diff
@@ -1636,32 +1550,21 @@ \subsubsection{Conversion rules}\label{conversion-rules}
 Behaviour in unusual cases:
 \begin{itemize}
 \item
-  if NewModifier list is empty then the modifier is just removed
+  if \lstinline!NewModifier! list is empty then the modifier is just removed
 \item
-  If OldModifer list is empty it is added for all uses of the class
+  If \lstinline!OldModifer! list is empty it is added for all uses of the class
 \item
-  If OldModifier\_i is cardinality(a)=0 the conversion will only be
-  applied for a component comp if there are no inside connections to
-  comp.a. This can be combined with other modifiers that are handled in
-  the usual way.
+  If \lstinline!OldModifier$i$! is \lstinline!cardinality(a) = 0! the conversion will only be applied for a component comp if there are no inside connections to \lstinline!comp.a!. This can be combined with other modifiers that are handled in the usual way.
 \item
-  If OldModifier\_i is cardinality(a)=1 the conversion will only be
-  applied for a component comp if there are any inside connections to
-  comp.a.
+  If \lstinline!OldModifier$i$! is \lstinline!cardinality(a) = 1! the conversion will only be applied for a component comp if there are any inside connections to \lstinline!comp.a!.
 \end{itemize}
 
-The converted modifiers and existing modifiers are merged such that the existing modifiers take precedence over the result of convertModifiers.
-A diagnostics is recommended if this merging removes some modifiers unless those modifiers are identical or it is the special case of an empty OldModifier list.
+The converted modifiers and existing modifiers are merged such that the existing modifiers take precedence over the result of \lstinline!convertModifiers!.  A diagnostics is recommended if this merging removes some modifiers unless those modifiers are identical or it is the special case of an empty OldModifier list.
```

Comment:
Missed OldModifier

Reply:
Fixed, together with some other nearby things.  In particular, I noted that the `\paragraph*` command needs some other actual paragraph content than a listing – otherwise the top of the box around the listing ends up on the same line as the paragraph title.

Please mark as resolved if the fix is OK.